Doc/library/zipfile.rst

```diff
@@ -368,7 +368,7 @@ ZipFile Objects
 
 
 .. method:: ZipFile.write(filename, arcname=None, compress_type=None, \
-                          compresslevel=None)
+                          compresslevel=None, strict_timestamps=True)
```

Comment:
I would prefer to use a keyword-only parameter. Same remark for other methods and functions.

Reply:
Bikeshedding: why not just "strict_timestamp=True" (no S)?

Reply:
there are more timestamps in the zip archive. each file has it's own.